api/app.js

```diff
@@ -2,12 +2,25 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var policies = require('./lib/routes/policies');
 var logger = require('./lib/log/logger');
+var HttpStatus = require('http-status-codes');
 var app = express();
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use('/v1/apps',policies);
 
+app.use(function(err, req, res, next) {
+  var errorResponse = {};
+  if (err) {
+    errorResponse = {
+      'success': false,
+      'error': err,
+      'result': null
+    };
+  }
+  res.status(HttpStatus.BAD_REQUEST).send(errorResponse);
```

Comment:
It is just weird to do a get /foo/bar and receive a 400 instead of a 404 or something more common.